applications/portal/frontend/src/constants/constants.ts

```diff
@@ -1,3 +1,12 @@
 // tab names at Home view
 export const ALLRESULTS = "all results";
 export const MYSUBMISSIONS = "my submissions";
+
+
+export const GET_ANTIBODY_TYPES = {
```

Comment:
There are a few problems with this variable name: first it's a verb, and variables shouldn't be verbs. Second, this is not antibody types, it's more about the current selection/search. A good name could be SEARCH_MODES, LISTING_STATES, something like that